### PR TITLE
[DBAAS-1254] Rebrand RHODA to OpenShift Database Access

### PR DESCRIPTION
**Pull request by jianrongzhang89**
Replace rhoda to dbaas.

### CODE REVIEW

File: console-extensions.json [hunk 1]
```diff
@@ -150,12 +150,12 @@
...
+      "id": "dbaas-admin-dashboard",
       "prefixNamespaced": true,
       "perspective": "admin",
       "section": "data-services",
       "name": "Database Access",
-      "href": "/rhoda-admin-dashboard"
+      "href": "/dbaas-admin-dashboard"
     }
   }
 ]
```
Comment (redhatHameed) [inline console-extensions.json:161]: add empty line here

Comment (jianrongzhang89) [reply console-extensions.json]: Done

**Review by redhatHameed — APPROVED** [review]
LGTM
wait for Tommy approval before merging thanks!

File: src/const.ts [hunk 2]
```diff
@@ -21,15 +21,15 @@ export const cockroachdbProviderName = 'CockroachDB Cloud'
 export const rdsProviderType = 'rds-registration'
 export const rdsProviderName = 'Amazon Relational Database Service'
 export const topologyInstructionPageUrl =
-  'https://access.redhat.com/documentation/en-us/red_hat_openshift_database_access/1/html/quick_start_guide/connecting-an-application-to-a-database-instance-using-the-topology-view_rhoda-qsg'
+  'https://access.redhat.com/documentation/en-us/red_hat_openshift_database_access/1/html/quick_start_guide/connecting-an-application-to-a-database-instance-using-the-topology-view_dbaas-qsg'
```
Comment (tchughesiv) [inline src/const.ts:24]: i expect all of these docs links will completely change... maybe to a new github based docs site?

**Review by tchughesiv — APPROVED** [review]
lgtm

**Pull request merged**